tests/decorators/decorators.go

```diff
@@ -39,4 +39,5 @@ var (
 	Istio                     = []interface{}{Label("Istio")}
 	InPlaceHotplugNICs        = []interface{}{Label("in-place-hotplug-NICs")}
 	MigrationBasedHotplugNICs = []interface{}{Label("migration-based-hotplug-NICs")}
+	LowCountAlerts            = []interface{}{Label("low-count-alerts")}
```

Comment:
Maybe "single-node" would be more generic?

Reply:
If I can chime in, labels can be descriptive: `RequiresTwoSchedulableNodes`

Reply:
+1

Reply:
I like it. Updated